hawkbit-mgmt/hawkbit-mgmt-resource/src/main/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtActionResource.java

```diff
@@ -71,6 +75,27 @@ public ResponseEntity<MgmtAction> getAction(final Long actionId) {
         return ResponseEntity.ok(MgmtActionMapper.toResponse(action, MgmtRepresentationMode.FULL));
     }
 
+    @Override
+    @AuditLog(entity = "Actions",type = AuditLog.Type.DELETE, description = "Delete Action", logResponse = true)
+    public ResponseEntity<Void> deleteAction(Long actionId) {
+        deploymentManagement.deleteAction(actionId);
+        return ResponseEntity.ok().build();
+    }
+
+    @Override
+    @AuditLog(entity = "Actions", type = AuditLog.Type.DELETE, description = "Delete Actions", logResponse = true)
+    public ResponseEntity<Void> deleteActions(String rsqlParam, List<Long> actionIds) {
+        if (!ObjectUtils.isEmpty(rsqlParam)) {
+            deploymentManagement.deleteActionsByRsql(rsqlParam);
```

Comment:
If !ObjectUtils.isEmpty(actionIds) throw exception.
If user has specified both - maybe he hasn't understood the method behavior - he may expect that both specified sets (or intersection) will be deleted